config/cscs.py

```diff
@@ -1000,7 +1000,7 @@
                 '--perflogdir=$APPS/UES/$USER/regression/maintenance/logs',
                 '--stage=$SCRATCH/regression/maintenance/stage',
                 '--report-file=$APPS/UES/$USER/regression/maintenance/reports/maint_report_{sessionid}.json',
-                '--reservation=maintenance',
+                '-Jreservation=maintenance',
```

Comment:
```suggestion
                '-J reservation=maintenance',
```

And please change it also in all other places.

Reply:
If I add the space I do not get the correct job script. It contains a space between the `--` and the `reservation`.

I ran reframe using

```
$ reframe -C cscs.py -n Gromacs --mode=maintenance -r
```

The created  job script  was
```
#!/bin/bash                                                                        
#SBATCH --job-name="rfm_GromacsGPUCheck_small_maint_job"                           
#SBATCH --ntasks=72                                                                
#SBATCH --ntasks-per-node=12                                                       
#SBATCH --output=rfm_GromacsGPUCheck_small_maint_job.out                           
#SBATCH --error=rfm_GromacsGPUCheck_small_maint_job.err                            
#SBATCH --time=0:10:0                                                              
#SBATCH --account=csstaff                                                          
#SBATCH --constraint=gpu                                                           
#SBATCH --hint=nomultithread                                                       
#SBATCH --switches=1                                                               
#SBATCH -- reservation=maintenance                                                 
module load daint-gpu                                                              
module load GROMACS                                                                
export CRAY_CUDA_MPS=1                                                             
srun gmx_mpi mdrun -dlb yes -ntomp 1 -npme 0 -s herflat.tpr 
```

Reply:
Indeed, this is a bug! Can you open a bug report?